c2rust-analyze/src/main.rs

```diff
@@ -449,13 +449,65 @@ fn run(tcx: TyCtxt) {
             assert_eq!(local, l);
         }
 
+        // Find all calls to `malloc`, `calloc`, `realloc`, and `free`
+        // and track their destination locals as being libc::c_void
```

Comment:
I think we should just call it `c_void` in general, as it could be `core::ffi::c_void` or `libc::c_void` depending on the Rust version, and `c_void` seems clear enough on its own.

Reply:
addressed in cc90e3f7